app/frontend/components/pages/common.tsx

```diff
@@ -16,3 +21,17 @@ export const EpochDateTime = ({
     </span>
   )
 }
+
+export const BitBox02MultiAssetAlert = () => {
+  const cryptoProviderInfo = useSelector((state) => state.cryptoProviderInfo)
+  const isMultiAssetSupported = useIsWalletFeatureSupported(CryptoProviderFeature.MULTI_ASSET)
+  return !isMultiAssetSupported && cryptoProviderInfo?.type === CryptoProviderType.BITBOX02 ? (
+    <Alert alertType="warning">
+      BitBox02 currently does not support sending Cardano tokens. If you need to send/receive
+      tokens, wait for the firmware update or use Ledger instead.{' '}
+      {
+        // TODO add link to more info
```

Comment:
planning to add bitbox tutorial once done/published by them

Reply:
Please link to this guide: 

https://shiftcrypto.support/help/en-us/35-adalite-cardano

We will flesh it out some more, but the link will stay the same.